lib/cocoapods-core/specification.rb

```diff
@@ -58,6 +58,10 @@ def initialize(parent = nil, name = nil, test_specification = false)
     #
     attr_accessor :subspecs
 
+    # @return [Boolean] whether this is a test specification or not.
+    #
+    attr_accessor :test_specification
```

Comment:
@orta @DanToml I am not a big fan of exposing this but I need it when we load from json to get set. Will look for a better way.